### ISSUE TITLE
How to train or evaluate

### ISSUE THREAD

**Issue by wolf2127** [Amir-Samadi/SAFE-RL#2]
I would like to know how to be able to train or test your code？
When I try to run SAFE_RL.ipynb, it prompts me with
FileNotFoundError: [Errno 2] No such file or directory: 'teacher_models/A2C_pong_teacher_model.zip'
I think this is caused by the lack of training.
![选区_003](https://github.com/Amir-Samadi/SAFE-RL/assets/116620093/85d48599-40f2-4abc-993b-b03492f448f8)

**Comment by Amir-Samadi** [comment]
Yes, you should first train the teacher model (the black-box), then generate the dataset within the code, and finally train the SAFE-RL model. For your convenience, I have just released the pretrained teacher and SAFE-RL models for some DRL agents across various environments. You only need to generate the dataset now. Please check the Releases repository for instructions.
